_config.yml

```diff
@@ -447,3 +447,12 @@ apps:
   - gh: "Medical-Image-Analysis-Laboratory/mialsuperresolutiontoolkit"
     status: active
     dh: "sebastientourbier/mialsuperresolutiontoolkit"
+
+  - gh: "djarecka/fmriprep-fake"
+    status: active
+    dh: "djarecka/fmriprep_fake"
+    ds_type:
```

Comment:
can specify that there is no CI for this app: 

see template here

https://github.com/bids-apps/bids-apps.github.io/blob/89c84714819c787d2e7245d478693f7b6c190c25/_config.yml#L78